src/main/java/net/coderbot/iris/pipeline/patcher/AttributeShaderTransformer.java

```diff
@@ -2,8 +2,7 @@
 
 import net.coderbot.iris.gbuffer_overrides.matching.InputAvailability;
 import net.coderbot.iris.gl.shader.ShaderType;
-import net.coderbot.iris.shaderpack.transform.StringTransformations;
-import net.coderbot.iris.shaderpack.transform.Transformations;
+import net.coderbot.iris.shaderpack.transform.*;
```

Comment:
No star imports please

Reply:
I'll convert them back. My auto formatter does this, I can configure it.